lib/SimplePie/SimplePie.php

```diff
@@ -1559,6 +1596,7 @@ public function init()
 			{
 				// Create new parser
 				$parser = $this->registry->create('Parser');
+				$utf8_data = self::stripInvalidUtf8($utf8_data);
```

Comment:
Perhaps that should be something more along these lines? Pinging @Alkarex 
```suggestion
				if ($this->force_feed) {
					$utf8_data = self::stripInvalidUtf8($utf8_data);
				}
```